ex03.exs

```diff
@@ -62,7 +62,24 @@ defmodule Ex03 do
   """
 
   def pmap(collection, process_count, function) do
-    « your code here »
+    collection
+    |> split(process_count)
+    |> Enum.map(&chunk_task(&1, function))
+    |> Enum.concat
+  end
+
+  defp split(collection, process_count) do
+    chunk_size = Enum.count(collection) / process_count
+      |> Float.ceil
+      |> Kernel.round
+
+    collection
+    |> Enum.chunk_every(chunk_size)
+  end
+
```

Comment:
All was good up to this function. 

Here you correctly create a task, which can run in parallel.But then you wait for it to finish before returning. This means you only ever have one task running at a time, so there's no concurrency in your code.

### -5